lib/screens/mitgliedsliste/mitglied_details.dart

```diff
@@ -205,72 +199,102 @@ class MitgliedDetailState extends State<MitgliedDetail>
+              ),
+              MapWidget(
+                homeLocation: homeLocation,
+                memberAddress: memberAddress,
+              ),
+            ],
+          ),
+        ),
+      ],
+    );
   }
 
-  Widget _buildGeburtsdatum() {
-    final int age = widget.mitglied.getAlterAm();
-    return _buildBox(<Widget>[
-      Text(
-          'Alter: $age (${widget.mitglied.geburtsDatum.day < 10 ? '0' : ''}${widget.mitglied.geburtsDatum.day}.${widget.mitglied.geburtsDatum.month < 10 ? '0' : ''}${widget.mitglied.geburtsDatum.month}.${widget.mitglied.geburtsDatum.year})',
-          style: const TextStyle(color: Colors.white, fontSize: 18)),
-    ]);
+  Widget _buildGeneralInfos() {
+    final mitglied = widget.mitglied;
+    final int age = mitglied.getAlterAm();
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Text(
+          "Allgemeine Informationen",
+          style: Theme.of(context).textTheme.titleMedium,
+        ),
+        _buildBox(
+          Column(
+            children: [
+              ListTile(
+                leading: const Icon(Icons.event),
+                subtitle: const Text('Alter'),
+                title: Text(
+                    "$age (${widget.mitglied.geburtsDatum.prettyPrint()})"),
+              ),
+              if (mitglied.email.isNotNullOrEmpty)
+                ListTile(
+                  leading: const Icon(Icons.email),
+                  subtitle: const Text('E-Mail'),
+                  title: _buildLinkText("mailto", mitglied.email!),
+                ),
+              if (mitglied.emailVertretungsberechtigter.isNotNullOrEmpty)
+                ListTile(
+                  leading: const Icon(Icons.email),
+                  subtitle: const Text('E-Mail Vertretungsberechtigter'),
+                  title: _buildLinkText(
+                    "mailto",
+                    mitglied.emailVertretungsberechtigter!,
+                  ),
+                ),
+              if (mitglied.telefon1.isNotNullOrEmpty)
+                ListTile(
+                  leading: const Icon(Icons.call),
+                  subtitle: const Text('Festnetznummer'),
+                  title: _buildLinkText("tel", mitglied.telefon1!),
+                ),
+              if (mitglied.telefon2.isNotNullOrEmpty)
+                ListTile(
+                  leading: const Icon(Icons.call),
+                  subtitle: const Text('Mobilfunknummer'),
+                  title: _buildLinkText("tel", mitglied.telefon2!),
+                ),
+              if (mitglied.telefon3.isNotNullOrEmpty)
+                ListTile(
+                  leading: const Icon(Icons.call),
+                  subtitle: const Text('Geschäftlich'),
+                  title: _buildLinkText("tel", mitglied.telefon3!),
+                ),
+              ListTile(
+                leading: const Icon(Icons.tag),
+                subtitle: const Text('NaMi ID'),
```

Comment:
Die Nummer ist als Mitgliedsnummer bekannt und wird ja auch für den Login als solche verwendet. Technisch wissen wir, dass es zu Verwirrung führen könnte

```suggestion
                subtitle: const Text('NaMi Mitgliedsnummer'),
```